polkadot/node/network/statement-distribution/src/vstaging/mod.rs

```diff
@@ -605,6 +618,7 @@ pub(crate) fn handle_deactivate_leaves(state: &mut State, leaves: &[Hash]) {
 	let relay_parents = state.implicit_view.all_allowed_relay_parents().collect::<HashSet<_>>();
 
 	// fast exit for no-op.
+	// TODO [now]: this looks suspicious.
```

Comment:
`deactivate_leaf` returns vec of hashes that were actually dropped
https://github.com/paritytech/polkadot-sdk/blob/09503b1d2d844d0d4f89ea936689a303feaf40da/polkadot/node/subsystem-util/src/backing_implicit_view.rs#L162

So it could be refactored similar to
https://github.com/paritytech/polkadot-sdk/blob/09503b1d2d844d0d4f89ea936689a303feaf40da/polkadot/node/network/collator-protocol/src/collator_side/mod.rs#L1240-L1251